src/js/api/config.js

```diff
@@ -115,8 +66,10 @@ const Config = function(options, persisted) {
 
     _deserialize(allOptions);
 
-    allOptions.localization = Object.assign({}, Defaults.localization, allOptions.localization);
-    allOptions.localization.errors = Object.assign({}, Defaults.localization.errors, allOptions.localization.errors);
+    const language = getLanguage();
+    const { localization, intl } = allOptions;
+    const customLocalization = getCustomLocalization(localization, intl, language);
+    allOptions.localization = applyTranslation({}, customLocalization);
```

Comment:
In `setup-steps` we call `applyTranslation` only if we load a json translation. We have to re-do the translation process because the customizations specified by the developer take precedence over the translation json, which takes precedence over the defaults.

Reply:
Yes, but you also call `getCustomLocalization` before that which you've already done here.